src/pages/Admin/Admin_v2.tsx

```diff
@@ -140,15 +139,13 @@ const Admin_v2: React.FC = () => {
   };
 
   return (
-    <>
-      <S.Wrapper hasSelectedVenue={!!selectedVenue}>
-        {selectedVenue && (
-          <AdminSidebar
-            sidebarOptions={sidebarOptions}
-            selected={selectedOption}
-            onClick={setSelectedOption}
-          />
-        )}
+    <WithNavigationBar fullscreen>
```

Comment:
This is good for admin v1, but how about admin v1? Presumably it should be an equally small change to add there?

Reply:
It was already added to admin v1, it was just checking if venue exists and if it doesn't it didn't render the navbar, I removed that check and now it renders properly.

Reply:
Awesome